src/Mod/Sandbox/Gui/CMakeLists.txt

```diff
@@ -15,11 +15,11 @@ include_directories(
     ${XercesC_INCLUDE_DIRS}
 )
 
-if(BUILD_QT5)
+if(FREECAD_QT_MAJOR_VERSION EQUAL 5)
     include_directories(
         ${Qt5Core_INCLUDE_DIRS}
     )
-else()
+elseif(FREECAD_QT_MAJOR_VERSION EQUAL 6)
     include_directories(
         ${QT_INCLUDE_DIR}
     )
```

Comment:
is this correct? in the previous state it would seem like the else content was for qt4 but now you've set it for qt6. Should it not be something like `Qt6Core_INCLUDE_DIRS`?